motion/index.scss

```
@@ -0,0 +1,2 @@
+@import '_motion';
```

I think we can drop the underscore.

```suggestion
@import 'motion';
@import 'animation';
```